tools/src/verdaccio/verdaccio.plugin.ts

```diff
@@ -54,6 +63,7 @@ export const createNodes: CreateNodes = [
             config,
             storage,
             preTargets,
+            deps: projectConfiguration.implicitDependencies,
```

Comment:
Ah, this is why they're all explicit. It's confusing the `implicitDependencies` now work differently than is usual for an Nx workspace (also only for e2e projects, which further complicates things) :/

The `@nx/devkit` has helper functions for listing all project dependencies, which could be used here instead of the `implicitDependencies` array from `project.json`. That would be much easier to maintain than having to think about listing all direct and indirect dependencies in `project.json`s.

Reply:
resolved over task dependencies